src/Backend/Facebook.php

```diff
@@ -42,11 +43,12 @@ public function getRequest($url)
      */
     public function extractCount(array $data)
     {
-        if (isset($data['data']) && isset($data['data'][0]) && isset($data['data'][0]['total_count'])) {
-            return $data['data'][0]['total_count'];
-        }
-        if (isset($data['share']) && isset($data['share']['share_count'])) {
-            return $data['share']['share_count'];
+        if (isset($data['engagement']['reaction_count'])
```

Comment:
You can combine this to `isset($foo, $bar, $baz)`

Reply:
@core23 You are right. Done.